dataclients/kubernetes/clusterclient.go

```diff
@@ -550,6 +554,40 @@ func (c *clusterClient) loadEndpointSlices() (map[definitions.ResourceID]*skippe
 			result[resID].Endpoints = append(result[resID].Endpoints, o)
 		}
 	}
+	return result
+}
+
+// LoadEndpointAddresses returns the list of all addresses for the given service using endpoints or endpointslices API.
```

Comment:
not exposed function -> loadEndpointAddresses

Reply:
Fixed